hera_qm/ant_metrics.py

```diff
@@ -431,3 +436,155 @@ def save_antenna_metrics(self, metricsJSONFilename):
 
         with open(metricsJSONFilename, 'w') as outfile:
             json.dump(allMetricsData, outfile, indent=4)
+
+# code for running ant_metrics on a file
+def get_metrics_OptionParser(method_name):
```

Comment:
My guess is some of these helper functions (get_metrics_OptionParser, get_pol, generate_fullpol_file_list) will be generally useful, not just for ant_metrics. Would they make sense in a hera_qm_utils module or in the __init__.py file?